open_event/static/js/admin/event/callForPaper.js

```diff
@@ -0,0 +1,23 @@
+$( document ).ready(function() {
+    var counter = 0;
+    $('.add-session-type').click(function (){
+        var row = "<div class='col-sm-12 row-session-type'>" +
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2680258337/issues/source?bid=3233864&fileBranchId=3324174#l4)